docker-assets/manageiq.conf

```diff
@@ -67,8 +67,8 @@ RewriteRule ^.*$ https://%{SERVER_NAME}%{REQUEST_URI} [L,R]
   RewriteCond %{REQUEST_URI} ^/ansibleapi
   RewriteCond %{REQUEST_URI} !(.*)/$
   RewriteRule ^(.*)$ $1/ [R,NE,L]
-  ProxyPass /ansibleapi/ http://localhost:54321/api/
-  ProxyPassReverse /ansibleapi/ http://localhost:54321/api/
+  ProxyPass /ansibleapi/ http://#{ANSIBLE_SERVICE_NAME}/api/
```

Comment:
Fixed.  I thought it was a requirement to expose the Ansible API externally.

@Fryguy or @gtanzillo Do you know if we need to expose this to the outside world as we did in VM's?